generators/client/templates/vue/src/main/webapp/app/core/jhi-navbar/jhi-navbar.component.ts.ejs

```diff
@@ -19,14 +19,12 @@ export default class JhiNavbar extends Vue {
     <%_ if (enableTranslation) { _%>
     this.principal().refreshTranslation(this.currentLanguage);
     <%_ } _%>
-    axios.get('management/info').then((res) => {
-      if (res.data && res.data.activeProfiles && res.data.activeProfiles.indexOf('swagger') > -1) {
-        this.swaggerEnabled = true;
-      }
-      if (res.data && res.data.activeProfiles && res.data.activeProfiles.indexOf('prod') > -1) {
-        this.inProduction = true;
-      }
-    });
+    if (this.activeProfiles.indexOf('swagger') > -1) {
+      this.swaggerEnabled = true;
+    }
+    if (this.activeProfiles.indexOf('prod') > -1) {
+      this.inProduction = true;
```

Comment:
Are you sure this is set properly ? Should not it be set using computed methods ?

Reply:
done